requirements-dev.txt

```diff
@@ -18,6 +18,7 @@ pypandoc
 ipython
 
 # Linter
+black==19.10b0; python_version >= "3.6"
```

Comment:
FYI, this file is being shared with conda of the CI too, which has a slightly different syntax comparing to the regular PIP.

Reply:
Yep... I was hopeful https://github.com/conda/conda-build/pull/1984 supported it, but I think it's for something different. :( I'll add it differently.

Reply:
Actually, I was trying to figure out yesterday where the other library requiring Python (Conda) 3.6+ is and couldn't find it; figured I'd support this similarly.

https://github.com/databricks/koalas/blob/master/.travis.yml#L69